python/tests/test_update_package_properties.py

```diff
@@ -0,0 +1,207 @@
+import pytest
+import os
+from shutil import copyfile
+
+from .test_utils import are_strings_equal
+from ..update_package_properties import *
+
+TEST_BASE_PATH = pathlib2.Path(__file__).parent.absolute()
+BASE_PATH = os.getenv("BASE_PATH", default=pathlib2.Path(__file__).parents[1])
+GITHUB_TOKEN = os.getenv("GH_TOKEN")
+PROJECT_VERSION = os.getenv("PROJECT_VERSION", default="10.0.3")
+TAG_NAME = os.getenv("TAG_NAME", default="v10.0.3")
+PROJECT_NAME = os.getenv("PROJECT_NAME", default="citus")
+MICROSOFT_EMAIL = os.getenv("MICROSOFT_EMAIL", default="gindibay@microsoft.com")
+NAME_SURNAME = os.getenv("NAME_SURNAME", default="Gurkan Indibay")
+CHANGELOG_DATE = datetime.strptime('Thu, 18 Mar 2021 01:40:08 +0000', '%a, %d %b %Y %H:%M:%S %z') if os.getenv(
+    "CHANGELOG_DATE") is None else datetime.strptime(os.getenv("CHANGELOG_DATE"), '%a, %d %b %Y %H:%M:%S %z')
+
+
+def get_default_changelog_param_for_test(latest_changelog, changelog_date):
+    changelog_param = ChangelogParams()
+    changelog_param.project_version = PROJECT_VERSION
+    changelog_param.project_name = PROJECT_NAME
+    changelog_param.microsoft_email = MICROSOFT_EMAIL
+    changelog_param.latest_changelog = latest_changelog
+    changelog_param.name_surname = NAME_SURNAME
+    changelog_param.fancy_version_number = 1
+    changelog_param.fancy = True
+    changelog_param.changelog_date = changelog_date
+    return changelog_param
+
+
+def test_get_version_number():
+    assert get_version_number("10.0.3", True, 1) == "10.0.3-1"
+
+
+def test_get_version_number_with_project_name():
+    assert get_version_number_with_project_name("citus", "10.0.3", True, 1) == "10.0.3.citus-1"
```

Comment:
This `citus` suffix to the version isn't always the package name. It's used as the repository name. For instance, for citus-enterprise packages, we still use `citus` as this suffix. https://github.com/citusdata/packaging/blob/all-enterprise/pkgvars

Reply:
Fixed